tests/braille-specs/he-int-g1.yaml

```diff
@@ -0,0 +1,21 @@
+display: |
+  space \x00a0 a
+  include spaces.uti
+  include unicode.dis
```

Comment:
This is a bit weird. I don't see any virtual dot "a" in this test. So why not just use unicode.dis? Also, including a non-display table (spaces.uti) in here is not recommended. This will probably stop working in the future.

Reply:
While creating YAML tests, I got confronted with some weird problems related to spaces.
Eg: Expected output ' ', got ' '.
Since they are not specially defined in the table, I thought that this is the best solution to avoid the problem, but as you said, I was wrong .

Reply:
This issue is most likely because of the difference between a normal space and a braille space (U+2800). If that happens you can use the unicode-without-blank.dis table.